pkg/kube/client.go

```diff
@@ -177,6 +188,12 @@ func (c *Client) Get(namespace string, reader io.Reader) (string, error) {
 		return "", err
 	}
 
+	//here, we will add the objPods to the objs
+	for key, podItems := range objPods {
+		for _, pod := range podItems {
+			objs[key+"(relation)"] = append(objs[key], &pod)
```

Comment:
This is a really good idea for distinguishing the related pods from default pods! One small change request here though. I think "related" or "children" would be a bit clearer than "relation" for a user. What do you think?

Reply:
I think it used "related" is better than "relation".